spring-xd-dirt/src/main/java/org/springframework/xd/dirt/module/store/ZooKeeperComposedModuleDefinitionRegistry.java

```diff
@@ -50,6 +53,13 @@
 
 	private final ModuleDependencyRepository moduleDependencyRepository;
 
+	/**
+	 * A reference to the main module registry (which also typically contains this registry as a delegate),
+	 * used to re-hydrate {@link org.springframework.xd.module.SimpleModuleDefinition}s with locations that
+	 * make sense on this container.
```

Comment:
Since this is not only `container` but also `admin`, we can say `admin/container`. I will fix while merging.